app/views/shared/_cancel_action_modal.html.slim

```diff
@@ -22,7 +22,10 @@
         - if idv
           = button_to t('idv.buttons.cancel'), verify_session_path, method: :delete,
             class: 'btn btn-outline blue border border-blue rounded-lg col-12 input-submit'
-        - else
+        - elsif sign_up_init
+          = button_to cancel_link_text, destroy_user_session_path, method: :get,
```

Comment:
Makes sense, I think I misunderstood the endpoint, I thought we would want to clear out the rails session as well. But if this is devise specific, we definitely don't need to call it.